lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnClusterSubmissionFromCS.java

```diff
@@ -83,6 +84,7 @@ private YarnClusterSubmissionFromCS(final AvroYarnAppSubmissionParameters yarnAp
     this.queue = DEFAULT_QUEUE;
     this.tokenKind = yarnClusterJobSubmissionParameters.getSecurityTokenKind().toString();
     this.tokenService = yarnClusterJobSubmissionParameters.getSecurityTokenService().toString();
+    this.fileSystemUrl = yarnJobSubmissionParameters.getFileSystemUrl().toString();
```

Comment:
now I am also suspicious about other `.toString()` calls in this code. 😄 could you please check if we need them, too?